tests/python/train/test_mlp_pruning.py

```diff
@@ -0,0 +1,86 @@
+# pylint: skip-file
+import mxnet as mx
```

Comment:
This is an example, not a test. Move SparseSGD and the code that runs it into the example folder